packages/calcite-components/src/utils/dynamicCSSConst.ts

```diff
@@ -0,0 +1,7 @@
+export const getWidth = (width: string, widthScale: string): string => {
```

Comment:
Can we rename this module to `dynamicClasses`? In the future, we could add a util for `scale`.